composer.json

```diff
@@ -25,7 +25,7 @@
         "jamesmoss/toml": "~0.1"
     },
     "autoload": {
-        "psr-0": { "Igorw\\Silex": "src" }
+        "psr-4": { "Igorw\\Silex\\": "src" }
```

Comment:
I'am not sure about this, since there is currently no problems at all mixing psr-0 and psr-4